.gitmodules

```diff
@@ -1,8 +1,3 @@
-[submodule "docs/themes/uswds"]
-	path = docs/themes/uswds
-	url = https://github.com/usnistgov/hugo-uswds.git
-	branch = master
-	update = rebase
 [submodule "build/metaschema"]
 	path = build/metaschema
 	url = https://github.com/usnistgov/metaschema.git
```

Comment:
Maybe blocking 🤷: we still need to figure out this piece (stick with this, move to metaschema-xslt in the same PR) before we can formally, merge, right?

Reply:
I'll take guidance here.  I'm not quite as confident on the best course.

Reply:
@nikitawootten-nist with https://github.com/usnistgov/metaschema-xslt/pull/45 on the way and close to setting off the baking timer in the oven, I assume we are close to ready to switch out this sprint?